Docs/source/developers/developers.rst

```diff
@@ -22,3 +22,5 @@ Our Doxygen API documentation in classic formatting `is located here <../_static
    testing
    documentation
    workflow
+
+See also the `WarpX tutorial slides <../_static/WarpX_tutorial.pdf>`_.
```

Comment:
I think the slides are so nice, they should be read before all that we are including in the developers section of the documentation. So I would maybe suggest putting this sentence with more details on what the PDF contains, even before mentioning the Doxygen API.